readme.md

```diff
@@ -79,34 +79,52 @@ From within the [ArcGIS Pro conda environment](http://pro.arcgis.com/en/pro-app/
 
 #### Development Usage
 
-1. `pip install .\` from the directory containing `setup.py`.
-1. from the `**/src**` directory execute `python -m forklift -h` for usage.
+- create new env
+  - `conda create --name forklift --clone arcgispro-py3`
+  - `activate forklift`
+- install deps
+  - `conda install flake8 mock`
+  - `pip install nose-cov rednose`
+- optionally install forklift
+  - `cd forklift`
+  - `pip install .\ -U`
+- run forklift
+  - for the installed version execute `forklift -h`
+  - for the source version, from the `**/src**` directory, execute `python -m forklift -h` for usage
 
 #### Tests
 
-On first run: `pip install tox`
-
-On subsequent runs: `tox`
+##### On first run
+- install deps
+  - `conda install flake8 mock`
+  - `pip install nose-cov rednose`
+- run tests
+  - `nosetests --with-id --rednose --cov-config .coveragerc --with-coverage --cover-package forklift --cov-report term-missing --cover-erase`
+  
+##### On subsequent runs
+`nosetests --with-id --rednose --cov-config .coveragerc --with-coverage --cover-package forklift --cov-report term-missing --cover-erase`
 
+_Tests that depend on a local SDE database (see `tests/data/UPDATE_TESTS.bak`) will automatically be skipped if it is not found on your system._
 
-Tests that depend on a local SDE database (see `tests/data/UPDATE_TESTS.bak`) will automatically be skipped if it is not found on your system.
+#### Linting
+`flake8`
 
-## Changelog
+# Changelog
 
-__8.2.0__
+**8.2.0**
```

Comment:
[I've felt this way for a while](https://github.com/agrc/gis.utah.gov#common-website-tips)